### DISCUSSION TITLE
Documentation on networking and sockets

### DISCUSSION THREAD

**Discussion by bonezinger** [vlang/v#9237] [General]
I cannot find anything on the docs about working with sockets and ports, dealing with incoming requests (https, telnet, tsl, etc), listening and/or sending reponses. Is networking implemented in any capacity in V or is still being worked on? If it's already implemented, where can I find documentation or examples of it?

**Comment by JalonSolov** [comment]
https://modules.vlang.io/net.html is a place to start.

**Reply by bonezinger** [reply]
Ah, I only looked at https://github.com/vlang/v/blob/master/doc/docs.md , I wasn't aware of that other link (vlib). Thanks
